packages/next-app/src/lib/gql.ts

```diff
@@ -34,8 +36,47 @@ const networkSdks = {
   [Network.goerli]: goerliSdk,
 };
 
+
+const mainnetMultisig = [
+  '0x7c68c42De679ffB0f16216154C996C354cF1161B',
+  '0xf4A80929163C5179Ca042E1B292F5EFBBE3D89e6',
+  '0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f',
+  '0xc38c5f97B34E175FFd35407fc91a937300E33860',
+  '0x75a52c0e32397A3FC0c052E2CeB3479802713Cf4',
+  '0x166f54F44F271407f24AA1BE415a730035637325',
+  '0x0EFcCBb9E2C09Ea29551879bd9Da32362b32fc89',
+]
+
+const arbitrumMultisig = [
+  '0x7c68c42De679ffB0f16216154C996C354cF1161B',
+  '0xc38c5f97B34E175FFd35407fc91a937300E33860',
+  '0xaF23DC5983230E9eEAf93280e312e57539D098D0',
+]
+
+const polygonMultisig = [
+  '0x7c68c42De679ffB0f16216154C996C354cF1161B',
+  '0xc38c5f97B34E175FFd35407fc91a937300E33860',
+  '0xeE071f4B516F69a1603dA393CdE8e76C40E5Be85',
+]
```

Comment:
not sure if we would be hardcoding all addresses here, would ask @mendesfabio, and I also know some of them have DAO members and some the Maxi as signers. I'll also ask the grants folks directly

Reply:
most of these addresses are related to other DAO operations. we only care about the ones that have permission to set swap fees: Mainnet Fee Setter (Eth) and Protocol Fees Multisig (Arb and Poly)

<img width="928" alt="image" src="https://user-images.githubusercontent.com/43360747/225915803-8bcb1fbb-f7a6-4e1a-8379-9e840b5cc102.png">